main.cmake

```diff
@@ -1,3 +1,6 @@
+set(MENDER_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}")
+set(MENDER_SRC_DIR "${CMAKE_CURRENT_SOURCE_DIR}/src")
```

Comment:
`MENDER_SOURCE_DIR` and `MENDER_SRC_DIR` are very similar. Could we call the former one `MENDER_TOPLEVEL_DIR`, `MENDER_ROOT_DIR` or something similar?

Reply:
With the changes after oleorhagen's comments, the variable was removed, so this is irrelevant now.